ePBFT/src/pbft/libbyz/Replica.cpp

```diff
@@ -253,20 +253,15 @@ bool Replica::compare_execution_results(
   return true;
 }
 
-size_t Replica::ledger_cursor() const
-{
-  return ledger_replay->cursor();
-}
-
 bool Replica::apply_ledger_data(const std::vector<uint8_t>& data)
```

Comment:
This could return an enum instead, a bit like `kv.h`'s `DeserialiseSuccess`. Values could be: `OK`, `ExecutionFailed`, `ExecutionResultInvalid`,

Reply:
I think boolean is OK for now, if we need a different return type later we can add it